cpp/include/cudf/concatenate.hpp

```diff
@@ -21,7 +21,17 @@
 #include <memory>
 #include <vector>
 
+/**
+ * @brief Column and Table APIs
+ * @defgroup column_apis Column and Table APIs
```

Comment:
Why call a group of column and table APIs just "column_apis"?

Reply:
Also strange to define the group for the first time in concatenate.hpp?

Reply:
> Why call a group of column and table APIs just "column_apis"?

The `column_apis` is just a reference name and is not shown anywhere in the output.